Package.swift

```diff
@@ -4,30 +4,23 @@ import PackageDescription
 
 let package = Package(
     name: "mParticle-Kochava",
-    platforms: [
-            .iOS("10.3"),
-            .tvOS("10.2"),
-        ],
+    platforms: [ .iOS("10.3") ],
     products: [
         .library(
             name: "mParticle-Kochava",
             targets: ["mParticle-Kochava"]),
     ],
     dependencies: [
-        .package(name: "mParticle-Apple-SDK", url: "git@github.com:mParticle/mparticle-apple-sdk.git", from: "8.2"),
-        .package(
-                    name: "KochavaCore",
-                    url: "https://github.com/Kochava/Apple-SwiftPackage-KochavaCore",
-                    from: "4.0.0"
-                ),
-        .package(name: "KochavaTracker", url: "https://github.com/Kochava/Apple-SwiftPackage-KochavaTracker.git", from: "4.0.0"),
+        .package(name: "mParticle-Apple-SDK",
+                 url: "https://github.com/mParticle/mparticle-apple-sdk",
+                 .upToNextMajor(from: "8.0.0")),
+        .package(name: "KochavaCore",
+                 url: "https://github.com/Kochava/Apple-SwiftPackage-KochavaCore", .upToNextMajor(from: "4.0.0")),
+        .package(name: "KochavaTracker", url: "https://github.com/Kochava/Apple-SwiftPackage-KochavaTracker.git", .upToNextMajor(from: "4.0.0")),
     ],
     targets: [
         .target(
             name: "mParticle-Kochava",
-            dependencies: ["mParticle-Apple-SDK","KochavaCore", "KochavaTracker"],
-            path: "mParticle-Kochava",
```

Comment:
Isn't this still needed? Or perhaps we should do as we did for others and move it under the sources directory. when trying to use your branch from xcode I get:

<img width="715" alt="Screen Shot 2021-05-04 at 3 21 16 PM" src="https://user-images.githubusercontent.com/1761038/117058354-8515f480-acec-11eb-9f85-c91a926af5d3.png">

Reply:
@samdozor Yep I pushed it up to test easier and got the same thing. Already re-added.